examples/bpi-monolith-test-suite/bpi/components/storage/agreement.ts

```diff
@@ -0,0 +1,75 @@
+import { Order } from "../../../domain-objects/order";
+
+export class Agreement {
+    productIds: string[] = [];
+    orders: Order[] = [];
+    proofs: string[] = [];
+    signature: boolean = false;
+
+    addOrder(order: Order): boolean {
```

Comment:
as a general practice let's use abstract naming and use identifiers to select which type of (commercial) document/transaction/state object is to be addressed ... this will allow us to reuse the code for all testing rather than very specific use cases. @ognjenkurtic